package.json

```diff
@@ -4,50 +4,52 @@
   "description": "IPFS Native Application",
```

Comment:
Let's take off the `1.0.0-alpha.1`, replace by `0.1.0` (that is a old pratice)

Reply:
Done!